spec/cqm/models_spec.rb

```diff
@@ -106,6 +110,8 @@
   end
 
   it 'shift patient data elements backwards in time' do
+    #     require 'byebug'
```

Comment:
These probably don't want to be in there

Reply:
Good catch. Removed.